.pylintrc

```
@@ -65,6 +65,7 @@ disable=abstract-method,
         coerce-builtin,
         coerce-method,
         delslice-method,
+        # disallowed-name, # copied from pyproject.toml
```

```suggestion
        # disallowed-name,
```
The comment probably adds little value after this PR (please also remove below)